plugins/nodes/src/js/components/NodeBreadcrumbs.js

```diff
@@ -19,22 +56,16 @@ const NodeBreadcrumbs = ({ nodeID, taskID, taskName, unitID }) => {
       </BreadcrumbTextContent>
     </Breadcrumb>
   ];
-  let node;
-
-  if (nodeID != null && trimmedNodeID.length > 0) {
-    node = CompositeState.getNodesList()
-      .filter({ ids: [nodeID] })
-      .last();
+  const node = useNode(nodeID);
```

Comment:
```suggestion
  const node = node != null && trimmedNodeID.length > 0 ? useNode(nodeID) : null;
```

If you do this you can avoid the redundant trim and null condition in getNodeFromSummary, as well as whatever other overhead comes with useNode(null)

Reply:
I tried to implement this change but got an error saying I can't use a hook in a conditional 😞

Reply:
of course. I just haven't memorized the rules yet!